### DISCUSSION TITLE
The largest repository

### DISCUSSION THREAD

**Discussion by kalined** [community/community#39746] [Repositories] [answered]
### Select Topic Area

Question

### Body

Good day,

Maybe somebody knows what is the largest public git repository? (>= 1GB) Would like to test my script on the large repo, but can't find the right one :) 

Thanks in advance,

**Comment by ossanna16** [comment]
Hi there @EdgeSan and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Jay-o-Way** [comment]
What do you mean with "THE script" and "the right one"?

**Reply by kalined** [reply]
Hi @Jay-o-Way, 

Thank you for your reply! By "the script" I mean my own python script which checks the size of the repository. "The right one" - large repo, which is over 1GB in size.

**Comment by jfullstackdev** [comment, marked as answer]
try this 

https://github.com/freeCodeCamp/freeCodeCamp

**Reply by kalined** [reply]
Thanks! :)
